list.rb

```diff
@@ -0,0 +1,18 @@
+require './my_enumerable'
```

Comment:
- [ ] Kindly implement this import with `require_relative`. According to ruby convention, `require` should be used for external files, like gems, while `require_relative` should be used for referring to files within your directory.